src/api/app/views/webui2/webui/project/list.html.haml

```diff
@@ -0,0 +1,51 @@
+:ruby
+  @pagetitle = 'Public Projects'
+.card
+  - if @important_projects.present?
+    .card-body
+      %h3 Main Projects
+      .table-responsive
+        %table.table.table-sm.table-striped.table-bordered
+          %thead
+            %tr
+              %th Name
+              %th Title
+          %tbody
+            - @important_projects.each do |project|
+              %tr
+                %td
+                  = link_to(project[0], action: :show, project: project[0])
+                %td
+                  #{project[1]}
+  .card-body
+    %h3= @pagetitle
+    .d-inline
+      %p
+        - if @show_all
+          = link_to(projects_path) do
+            %i.fas.fa-toggle-on.text-primary
+            Exclude #{::Configuration.unlisted_projects_filter_description}
+        - else
+          = link_to(projects_path(show_all: true)) do
+            %i.fas.fa-toggle-off.text-primary
+            Include #{::Configuration.unlisted_projects_filter_description}
+        - unless User.current.is_nobody?
+          %span.float-right
+            = link_to(new_project_path) do
+              %i.fas.fa-plus-circle.text-primary
+              Create new project
+    - cachekey = Digest::MD5.hexdigest(@projects.join)
+    - projecturl = project_show_path('REPLACEIT')
```

Comment:
`REPLACEIT` is a leftover???

Reply:
This is a pattern of mine to generate links in javascript by using a template generated from rails.

Reply:
We probably could use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead, but they are also not supported by IE. This sucks